### PR TITLE
add modest fade animation to `<FlashMessage>`

### PR DESCRIPTION
**Pull request by masuP9**
## Related URL

https://smarthr.atlassian.net/browse/SHRUI-379?atlOrigin=eyJpIjoiMmRmZDEwYjAwYWY3NDk5MWEzNjIxOGVjZDE1ZDRiM2IiLCJwIjoiaiJ9

[Understanding Success Criterion 2.3.3: Animation from Interactions](https://www.w3.org/WAI/WCAG21/Understanding/animation-from-interactions.html) [\[日本語\]](https://waic.jp/docs/WCAG21/Understanding/animation-from-interactions.html)

## Overview

`bounce` that is FlashMessage's default animation is easy to notice, but can sometimes be too much.

For example, message when auto save is completed animated many times while user are in form screen.
On mobile devices, it is much more noisy.

I add fade animation to FlashMessage as modest animation. 

## What I did

- add animation type `fade` and `none`

### Other refactoring

- add prefers-reduced-motion query
- add role property to use role that does not have `aria-live="assertive"`
- fix layout with long text

## Capture

fade in animation example

See Story: https://deploy-preview-1457--smarthr-ui.netlify.app/?path=/story/flashmessage--types

### CODE REVIEW

File: src/components/FlashMessage/FlashMessage.stories.tsx [hunk 1]
```diff
@@ -1,87 +1,144 @@
...
+            onClose={() => {
+              console.log('close')
+            }}
+          />
+        </li>
+      ))}
+      <li>
+        <FlashMessage
+          type="success"
+          visible={true}
```
Comment (AtsushiM) [inline src/components/FlashMessage/FlashMessage.stories.tsx:39]: (nits)
せっかくなので `fade` と `none` のアクションを設定しておくと良いかもしれません。

**Review by AtsushiM — DISMISSED** [review]
LGTM 🙏 
nitsを検討していただけると 🙇

**Review by negiandleek — APPROVED** [review]
LGTM 🎉

File: src/components/FlashMessage/FlashMessage.stories.tsx (hunk 1 above)
Comment (masuP9) [reply src/components/FlashMessage/FlashMessage.stories.tsx:40]: Storyで作りましたー。

**Review by AtsushiM — APPROVED** [review]
LGTM.

**Pull request merged**